generated/niscope/session.py

```diff
@@ -2302,6 +2302,21 @@ def fetch(self, num_samples, timeout=5.0):
         '''
         return self._fetch(num_samples, timeout)
 
+    def fetch_array(self, num_samples, timeout=5.0):
+        import array
+
+        vi_ctype = visatype.ViSession(self._vi)  # case 1
+        channel_list_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case 2
+        timeout_ctype = visatype.ViReal64(timeout)  # case 9
+        num_samples_ctype = visatype.ViInt32(num_samples)  # case 9
+        wfm_array = array.array("d", range(num_samples * self._actual_num_wfms()))
+        addr, count = wfm_array.buffer_info()
```

Comment:
According to [documentation](https://docs.python.org/3/library/array.html#array.array.buffer_info) this is to be avoided.